lib/src/server_messages.dart

```diff
@@ -270,3 +283,11 @@ class ErrorField {
     }
   }
 }
+
+class Notification {
+  final int processId;
```

Comment:
Capitalize the trailing `d`, e.g. `ID` to stay consistent with the `Connection._processID`